lib/gui/app/theme.ts

```diff
@@ -67,8 +70,7 @@ export const colors = {
 
 const font = 'SourceSansPro';
 
-export const theme = {
-	colors,
+export const theme = _.merge({}, Theme, {
```

Comment:
You don't need lodash here: `const theme = { ...Theme, otherStuff }`

Reply:
properties that are objects are overwritten with the spread operator instead of being merged